tests/acme_diags/derivations/test_acme.py

```diff
@@ -89,17 +89,53 @@ def setUp(self):
         self.mock_file_axis: "FileAxis" = Mock()
         self.mock_file_axis.getData.return_value = np.arange(0, 1002)
 
```

Comment:
Hey @forsyth2, just an FYI that I overlooked these tests in review because the build was passing. 
I didn't realize it was throwing `TypeError: 'Mock' object is not subscriptable` since the build wasn't failing.
[Build with errors](https://github.com/E3SM-Project/e3sm_diags/runs/2688255484?check_suite_focus=true#step:7:19)

This issue and the test themselves have been fixed.